src/Model.ts

```diff
@@ -212,16 +219,6 @@ export default class Model extends Base {
     return this.withRequest('fetching', promise, abort)
   }
 
-  /**
-   * Merges old attributes with new ones.
-   * By default it doesn't merge arrays.
-   */
-  applyPatchChanges (oldAttributes: {}, changes: {}): {} {
-    return deepmerge(oldAttributes, changes, {
-      arrayMerge: dontMergeArrays
-    })
-  }
```

Comment:
Why you converted to a function from an instance method? Just curious

Reply:
because this does not have anything to do with the instance. It does not use `this` so it does not care about the instance, it's a pure function.